docs/modules/deployment/pages/core/containers.adoc

```diff
@@ -0,0 +1,94 @@
+
+[[container-deployment]]
+= Containerized Deployment
+:description: Learn how to install OpenNMS Horizon/Meridian in a containerized environment like Kubernetes or Red Hat OpenShift.
```

Comment:
Would it be better to say something like:

`Learn how to install OpenNMS Horizon/Meridian in a Kubernetes or Red Hat OpenShift environment`

Reply:
Yes. We have a variable I can use. I missed it here. Will update.

Reply:
Actually, this text is not included in the docs. It's descriptive text for search engines. Unfortunately, I can't use the variables here. But it never appears in the docs.